.github/workflows/ci.yml

```diff
@@ -292,29 +310,20 @@ jobs:
 
       - name: Provision & Test
         uses: nick-fields/retry@v3
+        env:
+          TEST_PLAN_NAME: ${{ matrix.plan }}
+          COMMIT_ID: ${{ steps.vars.outputs.sha_short }}
         with:
           timeout_minutes: 10
           max_attempts: 10
           command: cd e2e && pnpm start
           retry_wait_seconds: 30
+
+      - name: Ensure Deletion
+        working-directory: e2e
+        if: ${{ always() }}
```

Comment:
So this is the thing that allows the job to keep running even on cancelation ?

Reply:
yes, exactly